test/spec.js

```diff
@@ -2,6 +2,9 @@ import 'should';
 import wiki from '../src/wiki';
 import nock from 'nock';
 import fs from 'fs';
+// import rewire from 'rewire';
+
+const described = () => {};
```

Comment:
Yep, you can use 'describe.skip'

You can also use 'describe.only' to isolate a test

:)